app/controllers/users_controller.rb

```diff
@@ -19,11 +19,9 @@ def destroy
     redirect_to staff_dashboard_path
   end
 
-  def edit
-  end
+  def edit; end
```

Comment:
but FINE okay it's not a big deal, especially since we won't have to even have these in Rails 5 lol